### DISCUSSION TITLE
Something is wrong with task age calculation

### DISCUSSION THREAD

**Discussion by tiagommmelo** [GothenburgBitFactory/taskwarrior#3137] [Community support] [answered]
Hello community. 
A long time ago, I've changed some of the custom attributes of TaskWarrior to suit my needs, then I stopped using it for some time. Now I'm getting back to it and just noticed that something is wrong with how the age is being calculated:

![ageTaskWarrior](https://github.com/GothenburgBitFactory/taskwarrior/assets/84918106/7fdbc114-d49e-4a58-864a-1c6c586496c8)

As you can see, the age is being calculated getting this value **0.033** and multiplying it by 2.
I know that there's an easy way to fix the coefficient (2 in this example) by inserting the line below in the *.taskrc* file and some int or float value for 'x':

`urgency.age.coefficient=x`

By the way, my config is set with the value **2**

What I did in the past, however, was editing the value for the number that is multiplying it and I cannot find where I did it and how I actually changed that value (it should be something different than 0.033 I believe) as there's nothing like this in my .taskrc config file.

How could I change it back to default without having to reinstall or some other drastic measure, as I have other configurations on .taskrc file that I would not like to change?

Thanks in advance

**Comment by djmitche** [comment, marked as answer]
The `0.033` in this case is the age of the task, as a fraction of `urgency.age.max` both in days. So for example if `urgency.age.max` is 300 and the task is 10 days old, the "age" for purposes of urgency calculations is 10/300 = 0.033. With `urgency.age.coefficient = 2` that makes a total of 0.066 "urgency points" contributed by the task's age.

**Reply by tiagommmelo** [reply]
got it, it changes over time and it is not something I've altered by mistake, but the usual behavior of taskwarrior and, if I want to boost that, I should just change the `urgency.age.coefficient`. Thanks a lot for clarifying, @djmitche
